
pointcloud_process:
  manual_pointcloud_alignment_preprocess:
    min_cloud_size:       50
    # NOTE - task names in task_list below MUST match the subheadings under the tasks heading
    task_list:            [ clip, voxelize ]
    tasks:
      clip: 
        type:             clipping
        box:              [ -8, 8, -3, 3, -1, 3.7, 0, 0, 0, 0, 0, 0 ]
        keep_ordered:     true
        should_publish:   true
        publish_topic:    clipped_cloud
      voxelize:           
        type:             voxelizing
        leaf_sizes:       [ 0.03, 0.03, 0.03 ]
        keep_ordered:     false
        should_publish:   true
        publish_topic:    voxelized_cloud